xbmc/platform/darwin/DarwinUtils.mm

```diff
@@ -157,35 +165,24 @@
 {
   static bool ret = false;
   static std::once_flag flag;
-  std::call_once(flag, []
-  {
-    size_t path_size = 2*MAXPATHLEN;
-    char given_path[path_size];
-    memset(given_path, 0x0, path_size);
-    /* Get Application directory */
-    int result = GetExecutablePath(given_path, &path_size);
-    if (result == 0)
+  std::call_once(flag, [] {
+    auto executablePath = getExecutablePath();
+    auto sandboxPrefixPaths = {
+        // before iOS 8, probably can be removed entirely
+        @"/var/mobile/Applications/",
```

Comment:
As minimum iOS version is 9.0 which is higher then the below iOS 8, why not drop this element?

Reply:
yes, you have a point